lib/tessel/provision.js

```diff
@@ -258,6 +258,7 @@ util.inherits(AlreadyAuthenticatedError, Error);
 actions.AlreadyAuthenticatedError = AlreadyAuthenticatedError;
 
 module.exports = actions;
+module.exports.TESSEL_AUTH_KEY = filepath;
```

Comment:
@rwaldron it is in an other [PR](https://github.com/tessel/t2-cli/pull/349/files#diff-464d0e2fb3c59e8716aa5cacbb54b6b8R18) (forgot to remove it here)